Gems/Atom/RHI/Code/Source/RHI/Buffer.cpp

```diff
@@ -45,11 +45,6 @@ namespace AZ::RHI
 
     void Buffer::Shutdown()
     {
-        IterateObjects<DeviceBuffer>([]([[maybe_unused]] auto deviceIndex, auto deviceBuffer)
```

Comment:
Are these Shutdown calls moved somewhere else?

Reply:
They are called when the DeviceObjects are freed which is the case if they are not used elsewhere anymore by clearing the map here. The reason for this change is precisely that they can still be used in a command buffer in flight and will be released later. We tested this and everything works fine. The only explicit call outside of destruction we found is in the RPI StreamingImage and that still works as expected.